Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -29,6 +29,8 @@ namespace gdjs {
     absoluteCoordinates: boolean;
     /** Clear the previous render before the next draw? */
     clearBetweenFrames: boolean;
+    /**Quality of the antialiasing filter */
```

Comment:
Comments are sentences + Space before the start of the comment + `antialiasing` -> `anti-aliasing` + `quality` -> `type`
```suggestion
    /** The type of anti-aliasing to apply at rendering. */
```